Dockerfile-development

```diff
@@ -0,0 +1,23 @@
+FROM node:22-alpine
+
+RUN apk add --no-cache \
+  ffmpeg \
+  vips-tools \
+  perl
+
+WORKDIR /app
+
+ENV HOME=/data
```

Comment:
Why providing the env vars not in the compose files? It would be more flexible in a dev environment which is not baked into the image